CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+CHANGELOG
+=========
+
+1.1.0
+-----
+
+- Symfony 4 support
```

Comment:
`s/4/5` :) but technically it's not a version bump unless something changed?

Reply:
Oh seems like you did add the changelog in the 1.1.0 release but its not in the master branch
https://github.com/phpcr/phpcr-migrations/pull/9/commits

Reply:
surprising. not sure where the commit is coming from... can you add 1.1.1 for symfony 5? and also change the branch-alias in composer.json to say 1.1-x instead of 1.0-x?